frontend/src/FileBrowser.svelte

```diff
@@ -68,7 +68,7 @@
   {#if !dragging}
     <label class="filelabel">
       <slot />
-      <input type="file" bind:this="{input}" bind:files="{files}" />
+      <input type="file" multiple bind:this="{input}" bind:files="{files}" />
```

Comment:
I don't think we always want uses of the `FileBrowser` component to have multiple file selection enabled. Probably best to do this as a boolean component parameter. Then `multiple` can be true or false based on that variable.